save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ContestView.kt

```diff
@@ -69,6 +71,11 @@ external interface ContestViewState : State, HasSelectedMenu<ContestMenuBar> {
      * Flag that shows if current contest is featured or not
      */
     var isFeatured: Boolean
+
+    /**
+     * Contest
```

Comment:
Please indicate with a comment somewhere that this field is unused but acts as a marker of contest existence

Reply:
This field is now used to get the name of the contest in the view header

**+state.contest.name**